source/features/clean-conversation-headers.tsx

```diff
@@ -24,23 +25,22 @@ function initPR(): void {
 	observe('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header)', {
 		add(byline) {
 			byline.classList.add('rgh-clean-conversation-header');
-			const isMerged = select.exists('#partial-discussion-header [title="Status: Merged"]');
 			const isSameAuthor = select('.js-discussion > .TimelineItem:first-child .author')?.textContent === select('.author', byline)!.textContent;
 			const baseBranch = select('.commit-ref:not(.head-ref)', byline)!;
 			const isDefaultBranch = (baseBranch.firstElementChild as HTMLAnchorElement).pathname.split('/').length === 3;
 
 			// Removes: [octocat wants to merge 1 commit into] github:master from octocat:feature
 			// Removes: [octocat merged 1 commit into] master from feature
 			// Removes: octocat [merged 1 commit into] github:master from lovelycat:feature
-			for (const node of [...byline.childNodes].slice(isSameAuthor ? 0 : 2, isMerged ? 3 : 5)) {
+			for (const node of [...byline.childNodes].slice(isSameAuthor ? 0 : 2, isMergedPR() ? 3 : 5)) {
 				node.remove();
 			}
 
 			if (!isSameAuthor) {
 				byline.prepend('by ');
 			}
 
-			if (isDefaultBranch) {
+			if (isDefaultBranch || baseBranch.textContent!.endsWith(':master')) {
```

Comment:
This only works for the link I posted, but it should work for local PRs too. Don’t just drop the semicolon though, because `not-master` shouldn’t match.

Reply:
https://github.com/sindresorhus/refined-github/pull/3844

Reply:
`title` works